fern/docs/pages/docs/writing-content.mdx

```diff
@@ -49,6 +49,38 @@ navigation:
 ```
 </CodeBlock>
 
+### Nested Sections
+If you'd like a section to toggle into more sections and pages, you can nest sections within sections. Here's an example of how it's done.
+<CodeBlock title="docs.yml">
+```mdx
+navigation:
+  - tab: guides
+    layout:
+      - section: Learn
+        contents: 
+          - section: Key Concepts
```

Comment:
This is a longer example than necessary. Can you shorten it to just two pages in each section?